fendermint/vm/genesis/src/lib.rs

```diff
@@ -107,6 +120,11 @@ impl Collateral {
         Power(power.try_into().expect("clipped to u64::MAX"))
     }
 
+    /// Convert the collateral into u64 representation. Will cap at u64::MAX.
+    pub fn u64(&self) -> u64 {
+        self.0.atto().to_u64().unwrap_or(u64::MAX)
+    }
```

Comment:
What would be the possible purpose of this function? You can't cap atto to u64, it would be worth like less than $100. This is why the power scale was introduced.